airflow/www/app.py

```diff
@@ -200,6 +200,9 @@ def init_views(appbuilder):
             appbuilder.add_link("GitHub",
                                 href='https://github.com/apache/airflow',
                                 category="Docs")
+            appbuilder.add_link("REST API Reference",
+                                href='/api/v1./api/v1_swagger_ui_index',
+                                category="Docs")
```

Comment:
```suggestion
                                category="API Docs")
```

Reply:
This is a tab in the menu. We already have WWW and other resources in this tab, so it's okay for me.

Reply:
oh yes